Dockerfile

```diff
@@ -32,6 +32,10 @@ WORKDIR /build
 COPY go.mod go.sum ./
 RUN go mod download
 
+# Copy the tools mod file for better layer caching when building locally
+COPY tools/go.mod tools/go.sum ./tools/
+RUN cd tools && go mod download
```

Comment:
To make this happen we need an additional entry here: https://github.com/enterprise-contract/ec-cli/blob/d12edd162458b07ecbab38fb1ba3e08d248fc46f/.tekton/cli-main-ci-pull-request.yaml#L32-L33

Maybe we need another sub-module, or just add this to the one in `/`? Not sure if we want the SBOM of our image to include everything that is in under `tools`.

Reply:
Opted to do a separate golang module. TIL: we already have `tools/go.mod` and `acceptance/go.mod` in the SBOM, see #1924